src/codebook/data/makeFiltered.js

```diff
@@ -1,12 +1,13 @@
 export function makeFiltered(data, filters) {
   var filtered = data;
   filters.forEach(function(filter_d) {
+    console.log(filter_d);
     //remove the filtered values from the data based on the filters
     filtered = filtered.filter(function(rowData) {
       var currentValues = filter_d.values
         .filter(f => f.selected)
         .map(m => m.value);
-      return currentValues.indexOf(rowData[filter_d.value_col]) > -1;
+      return currentValues.indexOf("" + rowData[filter_d.value_col]) > -1;
```

Comment:
The filter values are character regardless of the data values because `d3.nest().key()` [coerces values to character](https://github.com/d3/d3-3.x-api-reference/blob/master/Arrays.md#nest_key).  So only the data retain numeric values.